scene/resources/2d/convex_polygon_shape_2d.cpp

```diff
@@ -65,7 +65,7 @@ bool is_convex(const Vector<Vector2> &p_points) {
 
 void ConvexPolygonShape2D::_update_shape() {
 	Vector<Vector2> final_points = points;
-	if (Geometry2D::is_polygon_clockwise(final_points)) { //needs to be counter clockwise
+	if (Geometry2D::is_polygon_clockwise(final_points)) { // Needs to be clockwise.
```

Comment:
The previous comment isn't super clear, but it could mean "if clockwise, make it counter clockwise".

Whereas your new comment could be taken to mean it is making it clockwise (when it is doing the opposite).

Reply:
I see what you mean, I hadn't noticed that when I was working on this in #80035. I'll drop the change.